src/main/java/Product.java

```diff
@@ -0,0 +1,36 @@
+import java.util.ArrayList;
+
+public class Product {
+    private String label;
+    private double price;
+
+    public String getLabel() {
+        return label;
+    }
+
+    public double getPrice() {
+        return price;
+    }
+
+    //Constructor for product with default values
+    Product(){
```

Comment:
I don't think having default values for a product is helpful

Reply:
The default values are coming from the condition of the task.